docs/configuration.md

```diff
@@ -283,6 +283,14 @@ Config file: `report-file-name`
 
 If HTML and/or JSON reporting is being used you can use this option to change the report file name.
 
+### `exclude-test-files-in-report` &lt;`flag`&gt;
+
+Default: `false`
+Command line: `exclude-test-files-in-report`
```

Comment:
I don't think this is needed as a command line option. Either your project is too large to be able to include unit test results, or it's not. That sounds like static config to me.